managed-services/transactional-email/reference-content/dmarc-configuration.mdx

```diff
@@ -0,0 +1,151 @@
+
+This documentation provides information about the mandatory and optional tags to configure when [adding a DMARC record](/managed-services/transactional-email/how-to/add-dmarc-record/) to your domain, what they mean, the expected value for them, and their default value. You will also find examples of DMARC configurations according to your use-case.
+
+## DMARC Policy
+
+DMARC policies specify how receiving email servers should handle messages that fail authentication checks (SPF and/or DKIM) for the domain.
+The three main DMARC policies are:
+
+- none: no action is taken by the receiving mail servers based on DMARC authentication results. This policy allows domain owners to assess the impact of DMARC enforcement on their email traffic without risking legitimate emails being blocked or sent to spam. It is used for monitoring and collecting data without impacting email delivery.
+
+- quarantine: suspicious emails that fail DMARC authentication are marked as spam or sent to a quarantine folder by the receiving mail server.
+
+- reject: emails that fail the DMARC authentication are rejected by the receiving mail server, and not delivered. This policy provides maximum protection against email spoofing or phishing attacks.
+
+<Message type="important">
+  Emails sent from sources not configured in your SPF record, or that do not have a DKIM **will be rejected if you use the `p=reject` policy**.
+</Message>
+
+
+| Tag     | Description                                                                               | Mandatory | Values expected                            | Default value |
+|---------|-------------------------------------------------------------------------------------------|-----------|--------------------------------------------|---------------|
+| `v`     | Indicates the DMARC version.                                                              | Yes       | `DMARC1`                                   | x             |
+| `p`     | Indicates the policy to apply when/if a check fails.                                      | Yes       | `none`, `quarantine`, or `reject`          | x             |
+| `sp`    | Indicates the subdomain policy to apply when/if a check fails.                            | No        | `none`, `quarantine`, or `reject`          | x             |
+| `adkim` | Defines the DKIM alignment mode.                                                          | No        | `r` (relaxed mode) or `s` (strict mode)    | `r`           |
+| `aspf`  | Defines the SPF alignment mode.                                                           | No        | `r` (relaxed mode) or `s` (strict mode)    | `r`           |
```

Comment:
Maybe we can add a small paragraph or note explaining what is "strict" and what is "relaxed" ?